base/range.jl

```diff
@@ -786,3 +786,5 @@ end
 
 in{T<:Integer}(x, r::Range{T}) = isinteger(x) && !isempty(r) && x>=minimum(r) && x<=maximum(r) && (mod(convert(T,x),step(r))-mod(first(r),step(r)) == 0)
 in(x::Char, r::Range{Char}) = !isempty(r) && x >= minimum(r) && x <= maximum(r) && (mod(Int(x) - Int(first(r)), step(r)) == 0)
+
+unique(x::Range) = x
```

Comment:
Isn't this a concern?
```
_unique(x::Range) = x
length(unique(linspace(0,0,10)))
1
length(_unique(linspace(0,0,10)))
10
```
I was under the impression that `Range`s are not in general unique.